lib/fog/storage/local/models/file.rb

```diff
@@ -55,7 +55,7 @@ def destroy
               break
             end
             pwd = Dir.pwd
-            if ::File.exist?(dir_path) && ::File.directory?(dir_path)
```

Comment:
`File.directory?` should already encapsulate `File.exist?` logic. Is there some nuance about which I'm unaware?